governance/program/src/state/proposal.rs

```diff
@@ -414,10 +417,12 @@ impl ProposalV2 {
     fn get_max_voter_weight_from_mint_supply(
         &mut self,
         realm_data: &RealmV2,
+        governing_token_mint: &Pubkey,
         governing_token_mint_supply: u64,
+        vote: Option<&Vote>,
```

Comment:
Same note here as in `resolve_max_voter_weight`, since it's silently ignored for the council mint, which is extra confusing.  Consider splitting this into `get_max_veto_voter_weight`, and removing the parameter from this one

Reply:
Fixed with `VoteKind`